test/langtools/jdk/javadoc/tool/testTransformer/TestTransformer.java

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug      8298405
+ * @summary  Markdown support in the standard doclet
+ * @library  /tools/lib ../../lib
+ * @modules jdk.compiler/com.sun.tools.javac.api
+ *          jdk.compiler/com.sun.tools.javac.main
+ *          jdk.javadoc/jdk.javadoc.internal.tool
+ * @build    toolbox.ToolBox javadoc.tester.*
+ * @run main TestTransformer
+ */
+
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.ServiceLoader;
+import java.util.stream.StreamSupport;
+
+import com.sun.source.doctree.DocCommentTree;
+import com.sun.source.doctree.DocTree;
+import com.sun.source.doctree.RawTextTree;
+import com.sun.source.doctree.TextTree;
+import com.sun.source.util.DocTreeFactory;
+import com.sun.source.util.DocTrees;
+
+import javadoc.tester.JavadocTester;
+import jdk.javadoc.doclet.DocletEnvironment;
+import jdk.javadoc.doclet.StandardDoclet;
+import toolbox.ToolBox;
+
+public class TestTransformer extends JavadocTester {
+    public static void main(String... args) throws Exception {
+        var tester = new TestTransformer();
+        tester.runTests();
+    }
+
+    ToolBox tb = new ToolBox();
+
+    @Test
+    public void testFindStandardTransformer_raw() throws Exception {
```

Comment:
Checked exceptions are not thrown:
```suggestion
    public void testFindStandardTransformer_raw() {
```

Reply:
I might be mistaken, but this and the testFindStandardTransformer_stream methods look like we are testing ServiceLoader API. I would leave just the stream version.

Reply:
While these are weak tests, since they exercise different code patterns for access the standard transformer, such as may subsequently be included in any documentation.  I would prefer to leave both tests in place.

I have removed the unnecessary `throws` clauses.